.github/workflows/closed.yml

```diff
@@ -11,6 +11,45 @@ env:
   STORYBOOK_AMPLIFY_ID: d17ja0ul7nrdy0
 
 jobs:
+
+  check-change-type:
```

Comment:
This job has been copied from `ci.yml`

Reply:
Is there a way of pulling it out into a shared file (if so, can create a ticket to tidy this up?)

Reply:
Yeah, we should be able to make use of reusable actions. I'll get a ticket created for this, as there's probably a few places we can make use of this!